src/Libraries/RevitServices/Transactions/TransactionManager.cs

```diff
@@ -199,19 +199,19 @@ public class AutomaticTransactionStrategy : ITransactionStrategy
     {
         public TransactionHandle EnsureInTransaction(TransactionWrapper wrapper, Document document)
         {
-            TransactionManager.Log("EnsureInTransaction - AUTO STRAT: Starting new Transaction");
+            TransactionManager.Log("EnsureInTransaction - AUTO START: Starting new Transaction");
```

Comment:
@Randy-Ma This string goes into the log. It's not visible in the UI. I'm leaving it un-localized.